README.md

```diff
@@ -144,5 +144,6 @@ Example to fetch 4 SettledDvPs and check if one of them has a payment of 8550000
         dvp -> dvp.paymentAmount == 8550000L,
         dvp -> dvp.paymentAmount == 4750000L);
 ```
+If the boolean argument (`exact`) is true, each incoming contract must match at least one predicate.
```

Comment:
Minor: could be continued: `otherwise it may skip several events until finds 2 which satisfy the conditions.`